packages/admin/src/styles/globals.scss

```diff
@@ -96,3 +96,16 @@ html {
 .tox-tbtn--enabled svg {
   fill: govuk-colour('white') !important;
 }
+
+.super-admin-link {
+  display: block;
+  white-space: pre;
+  position: absolute;
+  right: 0;
+}
+
+@media (max-width: 768px) {
```

Comment:
Can be defined inside the super-admin-link class

`.super-admin-link {
  display: block;
  white-space: pre;
  position: absolute;
  right: 0;
  @media (max-width: 768px) {
  display: none
  }
}`

Reply:
nice!